src/object/srv_obj_migrate.c

```diff
@@ -2703,6 +2703,16 @@ ds_migrate_stop(struct ds_pool *pool, unsigned int version)
 	if (tls->mpt_opc == RB_OP_REINT)
 		pool->sp_reintegrating--;
 	migrate_pool_tls_put(tls);
+	/* Wait for xstream 0 migrate ULT(migrate_ult) stop */
+	if (tls->mpt_ult_running) {
```

Comment:
seems no place set "tls->mpt_fini = 1" for XS 0? above "dss_thread_collective(migrate_fini_one_ult, &arg, 0);" did not do that for XS0.

Reply:
ah, right. Thanks.